src/main/resources/fabric.mod.json

```diff
@@ -37,5 +37,12 @@
 		"fabric-rendering-v1": "*",
 		"fabric-lifecycle-events-v1": "*",
 		"fabric-key-binding-api-v1": "*"
+	},
+
+	"custom": {
+		"loom:injected_interfaces": {
```

Comment:
I didn't know about this feature, but it looks very handy! Good to see what Loom can do these days.